db/wal_edit.h

```diff
@@ -150,6 +151,16 @@ class WalSet {
 
   const std::map<WalNumber, WalMetadata>& GetWals() const { return wals_; }
 
+  // Checks whether there are missing or corrupted WALs.
+  // Returns Status::OK if there is no missing nor corrupted WAL,
+  // otherwise returns Status::Corruption.
```

Comment:
The return value can be that of `GetFileSize()`, not just `Status::Corruption()`.